graph-commons/src/test/scala/ch/datascience/rdfstore/InMemoryRdfStore.scala

```diff
@@ -181,4 +181,12 @@ trait InMemoryRdfStore extends BeforeAndAfterAll with BeforeAndAfter {
   }
 
   protected def runQuery(query: String): IO[List[Map[String, String]]] = queryRunner.run(query)
+
+  protected def rdfStoreSize: Int =
```

Comment:
there is no jena API call for this?

Reply:
Maybe there is but I found it easier to do a simple query :)

Reply:
I just wonder if this gets expensive eventually

Reply:
That's just for the tests so I think it should be fine.